src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2343,6 +2355,11 @@ def webapp_up(cmd, name, location=None, sku=None, dryrun=False, logs=False, laun
             """ % (name, asp, rg_str, full_sku, os_val, location, src_path,
                    detected_version, runtime_version)
     create_json = json.loads(dry_run_str)
+    cmd.cli_ctx.config.set_value('defaults', 'group', rg_name)
```

Comment:
Does this get to the setting under the current folder or global `~/.azure`?
CC @limingu

Reply:
Thanks for pointing this out - with your original implementation this would always set in the directory its run from so if i am running from a local directory it would set it there. Now it doesn't seem to work at all. For this payload i am going remove this - i.e not use localcontext. Will update history as well

Reply:
See the config class details at: https://github.com/Microsoft/knack/blob/master/knack/config.py#L28

The use_local_config is default as None. That means the default is to use global instead of local confign

If you want to use local config, you should initialize config as use_local_config True or call set_to_use_local_config(True) to use the local config.

Then you can set_value or get from local config.

Reply:
@panchagnula, can you please try that out?

@limingu, we need a helper in CLI core that we can ensure the configuration gets to the local and then restore the `use_local_config` to the original value

Reply:
Yes, we can add the helper method for the local case.
For now, the following code will do that:
        use_local_config_original = cli_ctx.config.use_local_config
        cli_ctx.config.set_to_use_local_config(True)

        cli_ctx.config.get() or cli_ctx.config.set_value()

        cli_ctx.config.set_to_use_local_config(use_local_config_original)